src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -481,8 +477,31 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                                     INS_OPTS_NONE);
                     }
                 }
-            }
-            break;
+                break;
```

Comment:
Can you add an empty line after `break;`?

Reply:
sure, done.

---
In reply to: [417511496](https://github.com/dotnet/runtime/pull/35590#discussion_r417511496) [](ancestors = 417511496)